docs/contributor/resources/01-kyma.md

```diff
@@ -81,6 +81,18 @@ spec:
 > **CAUTION:**
 > Module referencing using NamespacedName and FQDN (Fully Qualified Domain Name) has been deprecated.
 
+### **.spec.modules[].managed**
+
+The **managed** field determines whether or not the Lifecycle Manager manages the module. By default, it is set to `true`. By setting it to `false`, the module is opted out of management by Lifecycle Manager. This means that the module and all its related resources remain in the remote cluster in the same state they were in when the module became unmanaged. From this time on, Lifecycle Manager does NOT reconcile the module and its resources. In addition, the labels `operator.kyma-project.io/managed-by=kyma` and `operator.kyma-project.io/watched-by=kyma` are removed from the module's resources. This may be relevant if you for instance use those labels as exclusion filters for a custom monitoring, for example using the Kyma Telemetry module.
+
+Once a module was successfully unmanaged, the module's status in **.status.modules[].state** shows `Unmanaged`. Once the `Unmanaged` state shows, the module's entry can also be deleted entirely from **.spec.modules[]**. As the module is already unmanaged, this still leaves the module and its related resources in the remote cluster.
```

Comment:
I think the last sentence ("even thogh") isn't what we want to say. We want to say that if the module is already unmanaged, the entry may also be removed entirely. Maybe something like:

```suggestion
To verify that a module was successfully unmanaged, check that the field **.status.modules[].state** has the status `Unmanaged`. Once the state is `Unmanaged`, the module's entry can be deleted from **.spec.modules[]**. This will only remove the module's entry in the KymaCR. The module and its related resources remain in the remote cluster.
```

Reply:
```suggestion
To verify that a module was successfully unmanaged, check that the field **.status.modules[].state** has the status `Unmanaged`. Once the state is `Unmanaged`, you can delete the module's entry from **.spec.modules[]** in the Kyma CR.  Nevertheless, the module and its related resources remain in the remote cluster.
```
Ok, how about this version?

Reply:
Thanks! LIke the suggestion. Deleting is optional and it actually doesn't make a difference if the unmanaged module is deleted from the KymaCR or not. It is just a cleanup possibility to only list things in KymaCR that are managed by us.